app/src/main/java/org/literacyapp/receiver/BootReceiver.java

```diff
@@ -73,11 +73,13 @@ public static void scheduleMergeSimilarStudents(Context context){
         JobInfo.Builder builderMergeSimilarStudents = new JobInfo.Builder(LiteracyApplication.MERGE_SIMILAR_STUDENTS_JOB_ID, componentNameMergeSimilarStudents);
         boolean requiresCharging = true;
         builderMergeSimilarStudents.setRequiresCharging(requiresCharging);
+        boolean requiresDeviceIdle = true;
+        builderMergeSimilarStudents.setRequiresDeviceIdle(requiresDeviceIdle);
```

Comment:
Will setting `requiresDeviceIdle` to `true` cause the job to be executed only once? Or repeatedly?